public/static/locales/de/donationLink.json

```diff
@@ -0,0 +1,20 @@
+{
+  "donationLinkTitle": "Erstelle deinen Spenden-Link",
+  "donationLinkDescription": "Gestalte den Spenden-Link so, dass er deinen Bedürfnissen entspricht, damit deine Spender eine reibungslose Ablauf erfahren. Verwende die Vorschau, um zu sehen, wie das Endergebnis aussehen wird.",
```

Comment:
Change to "Gestalte den Spenden-Link so, dass er deinen Bedürfnissen entspricht, damit deine Spender ein tolles Erlebnis haben. Verwende die Vorschau, um zu sehen, wie der Link am Ende aussehen wird."